script.js

```diff
@@ -0,0 +1,96 @@
+const API_URL = "https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&api_key=7d2a446aceb11d91875ece5f9a7f7d14&page=1";
+const IMG_PATH = "https://image.tmdb.org/t/p/w1280";
+const SEARCH_API = 'https://api.themoviedb.org/3/search/movie?api_key=7d2a446aceb11d91875ece5f9a7f7d14&query="'
+
+
+const form = document.getElementById("form");
+const searchMovie = document.getElementById("search");
+const main = document.getElementById("main");
+
+getMovies(API_URL);
+
+async function getMovies(url) {
+    try {
+        const response = await fetch(url);
+        const data = await response.json();
+
+        if (data.results) {
+            console.log(data.results);
+            showMovies(data.results);
+        } else {
+            console.log("No data");
+        }
+    } catch (error) {
+        console.error('Error:', error);
+    }
+}
+
+form.addEventListener("submit", (event) => {
+    event.preventDefault();
+
+    const searchKeyword = searchMovie.value;
+
+    if (searchKeyword && searchKeyword !== "") {
+        getMovies(SEARCH_API + searchKeyword) ;
+        searchMovie.value = "";
+    } else {
+        window.location.reload();
```

Comment:
Why do you want to reload the page if the form is submitted with no search input? I'd say do nothing in this case.

Reply:
I wanted the page to be refreshed if an empty value is entered, but now I understood that I don't need it :)